db/schema.rb

```diff
@@ -103,6 +106,7 @@
     t.string "slug", null: false
     t.boolean "enabled", default: false, null: false
     t.integer "broken_link_count", default: 0
+    t.boolean "unofficial", default: false
```

Comment:
I think this might be a hangover from a local commit in a different [draft PR](https://github.com/alphagov/local-links-manager/pull/1247/commits/5f32a20169a0382d1e5b25ea86a134d51738b834)

Reply:
Yeah - I thought I'd caught this slippery blighter when I was rebasing, but apparently it got past me again! Thanks!

Reply:
Fixed